### ISSUE TITLE
Scheda S2R

### ISSUE THREAD

**Issue by federicodemarco** [anticorruzione/npa#1926]
Buonasera,
chiedo gentilmente quali possano essere i casi d'uso nei quali utilizzare questa nuova scheda.
Capisco che si possa aggiungere o rimuovere un partecipante.
Ad esempio potrebbe servire se la PAD fa censire manualmente i partecipanti?
Grazie

**Comment by obigno77** [comment]
No, il caso d'uso è la variazione della composizione del partecipante prima dell'aggiudicazione.
Ovviamente potrà essere usata anche per correggere gli eventuali errori di trasmissione commessi in sede di primo impianto del sistema
